source/lib.civet

```diff
@@ -2401,6 +2418,26 @@ function processDeclarationConditionStatement(s: IfStatement | IterationStatemen
       block.expressions.push ["", s]
       s.parent = block
 
+function processDeclarations(statements: StatementTuple[]): void
+  gatherRecursiveAll statements, (n) =>
+    // @ts-ignore
+    n.type is "Declaration"
+  // @ts-ignore
+  .forEach ({ bindings }: DeclarationStatement) =>
+    bindings?.forEach (binding) =>
+      { initializer } := binding
+      return unless initializer
+
+      exp := initializer[2]
+      // Wrap thick pipeline 'x ||> f' => 'f(x),x' with parentheses when inside an initializer
+      if exp?.type is "PipelineExpression"
+        if exp.children.at(-2) is ","
```

Comment:
I'm not totally sure this condition is the right one... `exp.children.includes(",")` would be more intuitive to me. But perhaps it's guaranteed that each thing between commas is a single object, in which case this is correct.

Reply:
It's related to this line in `processPipelineExpression` where we push the comma then the return ref. https://github.com/DanielXMoore/Civet/blob/3115ff3fc694077bd8d6b3f5ffc4f56b8a0f26bf/source/lib.civet#L3221